packages/core/src/components/renvooi/renvooi.tsx

```diff
@@ -1,31 +1,72 @@
-import { Component, ComponentInterface, Fragment, FunctionalComponent, Prop, h } from "@stencil/core";
+import {
+  Component,
+  ComponentInterface,
+  Fragment,
+  FunctionalComponent,
+  Prop,
+  h,
+  Event,
+  EventEmitter,
+} from "@stencil/core";
 
-import { RenvooiValue } from "./renvooi.interfaces";
+import {
+  RenvooiMarkFunction,
+  RenvooiMarkItemHighlightEvent,
+  RenvooiRenderMarkFunction,
+  RenvooiValue,
+} from "./renvooi.interfaces";
 
 interface RenvooiRenderProps {
   value: RenvooiValue;
+  mark?: RenvooiRenderMarkFunction;
```

Comment:
Voor de fijneproevers:

```suggestion
  mark: RenvooiRenderMarkFunction | undefined;
```

Voor de complete werking van de helper functional component `<RenvooiRender />` is mag `mark` niet vergeten worden mee te geven. De optional property operator `?` betekent iets anders dan type `undefined`.

Reply:
Suggestie overgenomen.